### ISSUE TITLE
d3-ease build failure when using CJS in a Remix app

### ISSUE THREAD

**Issue by matt-aitken** [hipstersmoothie/react-window-splitter#17]
Our app is Remix and has `serverModuleFormat: "cjs",` in the `remix.config.js` file.

I get this error when trying to run the app using dev:

```
/project/sandbox/node_modules/@window-splitter/state/dist/commonjs/index.js:46
const easings = __importStar(require("d3-ease"));
                             ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /project/sandbox/node_modules/d3-ease/src/index.js from /project/sandbox/node_modules/@window-splitter/state/dist/commonjs/index.js not supported.
Instead change the require of /project/sandbox/node_modules/d3-ease/src/index.js in /project/sandbox/node_modules/@window-splitter/state/dist/commonjs/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/project/sandbox/node_modules/@window-splitter/state/dist/commonjs/index.js:46:30)
```

Here's a stripped back Remix app which reproduces this error: https://codesandbox.io/p/devbox/agitated-shape-jrndhh

**Comment by hipstersmoothie** [comment]
Odd, that package does almost nothing

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v0.2.3`](https://github.com/hipstersmoothie/react-window-splitter/releases/tag/v0.2.3) :rocket:

**Comment by matt-aitken** [comment]
Amazing thanks @hipstersmoothie 😃
